ui/src/components/dashboard/audio_list_item.vue

```diff
@@ -7,30 +7,40 @@
         a(href='javascript:;', @click='handle_edit_clicked').icon-link
           span.fa.fa-times(v-if='edit_mode')
           span.fa.fa-pencil(v-else=true)
-      .title
-        form.my-0.d-inline-block(
-          v-if='edit_mode'
-          @submit='handle_edit_url_submit'
-        )
+      .title(v-if='edit_mode')
+        form.my-0.d-block(@submit='handle_edit_submit')
+          input(
+            type='submit'
+            @click='handle_hidden_submit_click'
+            hidden='true'
+          )
           input.edit-url-input(
             v-model='edit_url_input_value'
             type='text'
             :disabled='loading'
             @click='handle_edit_url_input_click'
           )
-        a.public-link(
-          v-else=true
-          :title='public_link_title'
-          :href='audio.publicUrl'
-          target='_blank'
-        )
-          .basename
-            text-with-search-highlight(:text='basename', :q='q')
-          .extension(v-if='extension')
-            text-with-search-highlight(:text='extension', :q='q')
-          .no-extension(v-if='!extension') [no extension]
-        span.fa.fa-eye-slash(v-if='!audio.visible')
-      .description {{ display_description }}
+          input.edit-description-input(
+            v-model='edit_description_input_value'
+            type='text'
+            placeholder='Description'
+            :disabled='loading'
+            @click='handle_edit_description_input_click'
+          )
+      .audio-info(v-else=true)
```

Comment:
looks like `.audio-info` is unused?

Reply:
what about `div(v-else=true)`? class name is unneeded if nothing references it and it has no styles

Reply:
yeah, guess i don't like plain, naked divs haha
will modify that in a new commit

Reply:
it doesn't matter a whole lot, but there's a case to be made for only using css classes for what they're meant for (styling components) when they're needed and using plain divs for what they're meant for (organizing content).